st2common/bin/st2ctl

```diff
@@ -141,6 +141,7 @@ function clean_logs() {
 function getpids() {
   echo "##### st2 components status #####"
   COMPONENTS=${COMPONENTS/mistral/mistral-server mistral-api}
+  COMPONENTS=${COMPONENTS/st2chatops/hubot}
```

Comment:
This is no longer an issue with the new st2chatops package since it is not in a container.